kubernetes/samples/charts/traefik/README.md

```diff
@@ -4,102 +4,95 @@ This sample demonstrates how to install the Traefik ingress controller to provid
 load balancing for WebLogic clusters.
 
 ## Install the Traefik operator with a Helm chart
-The Traefik Helm chart is located in the official Helm project `charts` directory at https://github.com/helm/charts/tree/master/stable/traefik.
-The chart is in the default repository for Helm.
+This document is based on Traefik version 2.x with Helm chart at https://github.com/containous/traefik-helm-chart.
```

Comment:
make these links?

Reply:
This document is based on Traefik version 2.x with the Helm chart at [Traefik Repository](https://github.com/containous/traefik-helm-chart)